apps/web/app/(articles)/article/[slug]/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { headers } from "next/headers";
+import { dehydrate } from "@tanstack/query-core";
+import { Suspense } from "react";
+import getQueryClient from "../../../../lib/get-query-clinet";
+import { queryKeys } from "../../_constants/querykeys";
+import { getArticle } from "../../_api/get-article";
+import Hydrate from "../../../../lib/query-hydrate";
+import { ArticleDetail } from "../../_components/article-detail";
+
+export default async function ArticlePage(): Promise<JSX.Element> {
+  const headersList = headers();
+  const activePath = headersList.get("x-invoke-path");
+  const slug = activePath?.split("/")[2];
+
+  if (!slug) {
+    throw new Error("ERROR : article path is not found");
+  }
```

Comment:
```suggestion
type Params = {
  slug: string;
}

type ArticlePageProps = {
  params: Params;
}

export default async function ArticlePage({ params: { slug } }: ArticlePageProps): Promise<JSX.Element> {
```

페이지 컴포넌트는 props를 통해 params에 접근할 수 있습니다!

Reply:
감사합니다 이건 몰랐네요 덕분에 간결하게 작성할 수 있었어요

Reply:
[eat : header 에서 가져오는 params 를 page props 에서 가져올 수 있도록 수정합니다.](https://github.com/pagers-org/react-world/pull/132/commits/6b58141507c41a7cc0388723eab5f65c1592e010)